src/pages/Introduce/Introduce.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { useNavigate } from "react-router-dom";
+import "./introduce.scss";
+
+const Self = () => {
```

Comment:
```suggestion
const Introduce = () => {
```
컴포넌트의 이름은 파일 이름과 동일하게 구현해주세요!